pinax/stripe/tests/test_middleware.py

```diff
@@ -1,8 +1,11 @@
-from django.core.urlresolvers import reverse
+from django.contrib.auth import authenticate, login, logout, get_user_model
```

Comment:
We like to keep the `*.contrib` imports in their own section of imports after the other django imports.